ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -2,3 +2,46 @@
 # defaults file for installer
 cache_enabled: false
 provision_cache_store: "{{ ansible_env.HOME }}/rhcos_image_cache/"
+
+# Information used to create certificates when creating the registry.
+cert_country: MyCountry
+cert_state: MyState
+cert_locality: MyCity
+cert_organization: MyCompany
+cert_origanizational_unit: MyBusinessUnit
+
+# A file that contained extra auths to be included in the pull secret.
+disconnected_registry_auths_file:
+
+# Name of a file containing the additionalTrustBundle and mirrored
+#   repository settings. If this is set, then a local registry will not be created.
+#   Setting this causes the playbook to use an existing disconnected registry.
+disconnected_registry_mirrors_file:
+
+# The containers registry port
+registry_port_container: 5000
```

Comment:
Since right now we hardcode opening this specific port (5000), I don't think we should offer this value as something that is changeable. Instead I think this variable should exist in installer/vars/main.yml

Reply:
I add a task to open the correct port based upon the setting. Would it be better to remove the 5000 from the other tasks? Then the port will only open when disconnected is chosen

Reply:
So I agree what should happen is:
* Remove opening 5000 by default in our firewall
* Check if disconnected is enabled, if it is enabled open the port specified by the variable registry_port_container 
* We should add a variable inside inventory/hosts.sample that states
```
# Specify the tcp port to open for the disconnected registry container. Default vaue set to 5000. Uncomment to set a different port.
#registry_port_container=5000
```

Reply:
there are two vars used.
registry_port_container is the port the container uses.
registry_port is the port exposed on the host.
I think it would make more sense to modify the hosts.sample file for the registry_port variable since the registry_port_container setting will seldom change.

Reply:
Yeah that is what I meant. :)

Reply:
i added #registry_port=5000 to the hosts.sample to show user it can be changedpushed in matest commit